libs/api/domains/cms/src/lib/models/organizationNews.model.ts

```diff
@@ -0,0 +1,46 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+import { IOrganizationNews } from '../generated/contentfulTypes'
+import { Image, mapImage } from './image.model'
+import { mapOrganization, Organization } from './organization.model'
+
+@ObjectType()
+export class OrganizationNews {
+  @Field(() => ID)
+  id: string
+
+  @Field(() => Organization)
+  organization: Organization
+
+  @Field()
+  title: string
+
+  @Field({ nullable: true })
+  date?: string
+
+  @Field()
+  introduction: string
+
+  @Field({ nullable: true })
+  featuredImage?: Image
+
+  @Field({ nullable: true })
+  content?: string
+
+  @Field()
+  slug: string
+}
+
+export const mapOrganizationNews = ({
+  sys,
+  fields,
+}: IOrganizationNews): OrganizationNews => ({
+  id: sys.id,
+  organization: mapOrganization(fields.organization),
+  title: fields.title,
+  date: fields.date ?? '',
+  introduction: fields.introduction,
+  featuredImage: mapImage(fields.featuredImage),
+  content: (fields.content && JSON.stringify(fields.content)) ?? null,
+  slug: fields.slug,
+})
```

Comment:
We should maybe try and extend the default news model to support organizations. It would allow us to consolidate the news logic and maybe reuse some components/functionality

Reply:
Sounds good. We could add an `organization: Organization` field to the news model and make a small change to the web app so that only news entries with no organization are shown on the frontpage.

We also got a suggestion that it could be useful if important organization news could be shown on the frontpage, so maybe we should add something like an `important: boolean` field as well :)

Reply:
We already have tags on `news`under the `genericTags` field.
You can already query news by tags.
If we create an Organization tag we should not need to create any additional fields or code to fetch the news in each Organization page.
Additionally we might create an important tag but we might need to solve the leaky nested queries problem before using that. I believe it would be worth the work but definitely something we should only cross when we get to it

Reply:
I should also mention we are using the tags on the news overview atm so you can look at the logic there to see how to query by tag

Reply:
Thanks, we will look into using the article tags for this. We are going to remove the news from this pull request and add it in a separate PR, so I'm marking this as resolved here.